pkgs/by-name/ma/material-maker/package.nix

```diff
@@ -0,0 +1,161 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+  fetchurl,
+  nix-update-script,
+
+  # build deps
+  autoPatchelfHook,
+  copyDesktopItems,
+  makeDesktopItem,
+  undmg,
+
+  # runtime deps
+  libGL,
+  libGLU,
+  libX11,
+  libXcursor,
+  libXext,
+  libXfixes,
+  libXi,
+  libXinerama,
+  libXrandr,
+  libXrender,
+  alsa-lib,
+  libpulseaudio,
+  udev,
+}:
+let
+  pname = "material-maker";
+  version = "1.3";
+  versionUnderscore = "${lib.replaceStrings [ "." ] [ "_" ] version}";
+
+  sources = {
+    linux = fetchzip {
+      url = "https://github.com/RodZill4/material-maker/releases/download/${version}/material_maker_${versionUnderscore}_linux.tar.gz";
+      hash = "sha256-WEu5gVfnswB5zYzu3leOL+hKOBzJbn48gHQKshlfOh4=";
+    };
+    darwin = fetchurl {
+      url = "https://github.com/RodZill4/material-maker/releases/download/${version}/material_maker_${versionUnderscore}.dmg";
+      hash = "sha256-D4jPQEOKws2JFczh0K55mR08jIBMU6jOFOm6TyFxnt8=";
+    };
+  };
+in
+stdenvNoCC.mkDerivation (finalAttrs: {
+  inherit pname version;
+
+  src =
+    if stdenvNoCC.isLinux && stdenvNoCC.isx86_64 then
+      sources.linux
+    else if stdenvNoCC.isDarwin then
+      sources.darwin
+    else
+      throw "unsupported platform";
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    copyDesktopItems
+  ] ++ lib.optionals stdenvNoCC.isDarwin [ undmg ];
+
+  sourceRoot = ".";
+
+  buildInputs = [
+    libGL
+    libGLU
+    libX11
+    libXcursor
+    libXext
+    libXfixes
+    libXi
+    libXinerama
+    libXrandr
+    libXrender
+  ];
+
+  desktopItems = makeDesktopItem {
+    name = "material-maker";
+    exec = "material-maker";
+    icon = "material-maker";
+    genericName = "Procedural texture generation and 3D model painting tool";
+    desktopName = "Material Maker";
+    comment = finalAttrs.meta.description;
+    terminal = false;
+    categories = [
+      "Graphics"
+      "3DGraphics"
+    ];
+    keywords = [
+      "material"
+      "textures"
+      "paint"
+      "3d"
+    ];
+  };
+
+  unpackPhase = lib.mkIf stdenvNoCC.isDarwin ''
+    7z x $src
+  '';
```

Comment:
Build fails here. Isn't this supposed to be removed?

Reply:
Yes my bad seems to be a merge issue